werkzeug/http.py

```diff
@@ -374,7 +374,7 @@ def parse_options_header(value, multiple=False):
             return tuple(result)
         value = rest
 
-    return tuple(result)
+    return tuple(result) if result else ('', {})
```

Comment:
We already have a if-clause at the top of the function protecting against empty values, would be nice if that one could be fixed.